### PR TITLE
MBS 8128 Can pass APK to runner manually

### PR DESCRIPTION
**Pull request by eugene-krivobokov**

### CODE REVIEW

File: docs/content/docs/test/Runner.md [hunk 1]
```diff
@@ -15,7 +15,43 @@ It can do the following:
...
+extensions.getByType<GradleInstrumentationPluginConfiguration>().apply {
+    // all options goes here
+}
+
+```
+
+### Test APK
+
+Plugin builds APKs on his own by default.\
+If for any reason you have to build APK externally, you can pass files manually:
```
Comment (eugene-krivobokov) [inline docs/content/docs/test/Runner.md:41]: I see probable reasons to pass APK:
- It has been built externally on another host due to performance
- It has been transformed due to security reasons and we can't get a final artifact from AGP. Avito's signer service as an example.

File: subprojects/gradle/instrumentation-tests/src/main/kotlin/com/avito/instrumentation/InstrumentationTestsTask.kt [hunk 2]
```diff
@@ -65,9 +65,6 @@ abstract class InstrumentationTestsTask @Inject constructor(
     @Input
     val buildUrl = objects.property<String>()
 
-    @Input
-    val testedVariantName = objects.property<String>()
```
Comment (eugene-krivobokov) [inline subprojects/gradle/instrumentation-tests/src/main/kotlin/com/avito/instrumentation/InstrumentationTestsTask.kt:69]: Unused

File: subprojects/gradle/instrumentation-tests/src/main/kotlin/com/avito/instrumentation/InstrumentationTestsPlugin.kt [hunk 3]
```diff
@@ -254,9 +262,9 @@ class InstrumentationTestsPlugin : Plugin<Project> {
                                 }
 
                                 val isFullTestSuite = gitState.map {
-                                        it.isOnDefaultBranch
-                                                && instrumentationConfiguration.impactAnalysisPolicy is ImpactAnalysisPolicy.Off
-                                    }
+                                    it.isOnDefaultBranch
+                                        && instrumentationConfiguration.impactAnalysisPolicy is ImpactAnalysisPolicy.Off
```
Comment (matzuk) [inline subprojects/gradle/instrumentation-tests/src/main/kotlin/com/avito/instrumentation/InstrumentationTestsPlugin.kt:266]: hm, where is the impact analysis module in the general scheme of Avito runner?

**Comment by matzuk** [comment]
@eugene-krivobokov yep, it's a thing that we have discussed in an issue (83)

File: docs/content/docs/test/Runner.md (hunk 1 above)
Comment (dsvoronin) [reply docs/content/docs/test/Runner.md:41]: i think this comment could be a part of documentation

File: subprojects/gradle/instrumentation-tests/src/main/kotlin/com/avito/instrumentation/InstrumentationTestsPlugin.kt (hunk 3 above)
Comment (dsvoronin) [reply subprojects/gradle/instrumentation-tests/src/main/kotlin/com/avito/instrumentation/InstrumentationTestsPlugin.kt:266]: it is a part of the tests filtration; it is optional now, but we can possibly not use it as a dependency altogether

**Review by dsvoronin — APPROVED** [review]

File: docs/content/docs/test/Runner.md (hunk 1 above)
Comment (eugene-krivobokov) [reply docs/content/docs/test/Runner.md:41]: It is a documentation

**Pull request merged**